CitizenLabDotCo/citizenlab
# CL-1358: Add add error handling in survey builder

Adds error handling in survey builder.

## How to test
- Create a project with a survey
- Navigate to the Survey tab
- Click to edit the Survey
- Add a question type and leave it empty
- Try to save and see if that you get an error message in the UI

## Screenshots
![image](https://user-images.githubusercontent.com/12659000/188808238-d420c728-cabe-4be4-a1cd-2802a4fbbac0.png)



## How urgent is a code review?

- End of day if possible
